amlip_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+# CMake build rules for AML-IP Submodule
+###############################################################################
+cmake_minimum_required(VERSION 3.5)
+
+###############################################################################
+# Find package ddsrouter_cmake
+###############################################################################
+# Package ddsrouter_cmake is required to get every cmake macro needed
+find_package(ddsrouter_cmake REQUIRED)
```

Comment:
Should this cmake module be outside of DDS-Router repo?

Reply:
Indeed, it should :new_moon_with_face: